docs/io/unified_table_fits.rst

```diff
@@ -0,0 +1,625 @@
+  is no way to represent missing data.
+- The masked data values are permanently lost, precluding the possibility
+  of later unmasking the values.
+
+``astropy`` provides a work-around for this limitation that users can choose to
+use. The key part is to use the ``serialize_method='data_mask'`` keyword
+argument when writing the table. This tells the FITS writer to split each masked
+column into two separate columns, one for the data and one for the mask.
+When it gets read back that process is reversed and the two columns are
+merged back into one masked column.
+
+.. doctest-skip::
+
+  >>> from astropy.table.table_helpers import simple_table
+  >>> t = simple_table(masked=True)
+  >>> t['d'] = [False, False, True]
+  >>> t['d'].mask = [True, False, False]
+  >>> t
+  <Table masked=True length=3>
+    a      b     c     d
+  int64 float64 str1  bool
+  ----- ------- ---- -----
+     --     1.0    c    --
+      2     2.0   -- False
+      3      --    e  True
+
+.. doctest-skip::
+
+  >>> t.write('data.fits', serialize_method='data_mask', overwrite=True)
+  >>> Table.read('data.fits')
+  <Table masked=True length=3>
+    a      b      c      d
+  int64 float64 bytes1  bool
+  ----- ------- ------ -----
+     --     1.0      c    --
+      2     2.0     -- False
+      3      --      e  True
+
+.. warning:: This option goes outside of the established FITS standard for
+   representing missing data, so users should be careful about choosing this
+   option, especially if other (non-``astropy``) users will be reading the
+   file(s). Behind the scenes, ``astropy`` is converting the masked columns
+   into two distinct data and mask columns, then writing metadata into
+   ``COMMENT`` cards to allow reconstruction of the original data.
+
+``astropy`` Native Objects (Mixin Columns)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to store not only standard `~astropy.table.Column` objects to a
+FITS table HDU, but also any ``astropy`` native objects
+(:ref:`mixin_columns`) within a `~astropy.table.Table` or
+`~astropy.table.QTable`. This includes `~astropy.time.Time`,
+`~astropy.units.Quantity`, `~astropy.coordinates.SkyCoord`, and many others.
+
+In general, a mixin column may contain multiple data components as well as
+object attributes beyond the standard Column attributes like ``format`` or
+``description``. Abiding by the rules set by the FITS standard requires the
+mapping of these data components and object attributes to the appropriate FITS
+table columns and keywords. Thus, a well defined protocol has been developed
+to allow the storage of these mixin columns in FITS while allowing the object to
+"round-trip" through the file with no loss of data or attributes.
+
+Quantity
+~~~~~~~~
+
+A `~astropy.units.Quantity` mixin column in a `~astropy.table.QTable` is
+represented in a FITS table using the ``TUNITn`` FITS column keyword to
+incorporate the unit attribute of Quantity. For example:
+
+.. doctest-skip::
```

Comment:
I'll stop commenting on that, but all those doctests should be run and not be skipped.

Reply:
I was initially annoyed :smile:, but it seems to be going well and fixing a few mistakes along the way.